modules/core/src/main/java/org/apache/ignite/spi/deployment/local/LocalDeploymentSpi.java

```diff
@@ -69,6 +70,7 @@
 @IgnoreIfPeerClassLoadingDisabled
 public class LocalDeploymentSpi extends IgniteSpiAdapter implements DeploymentSpi {
     /** Enables additional check for resource name on resources removal. */
+    @SystemProperty(value = "Enables additional check for resource name on resources removal.")
```

Comment:
Enables an additional check of a resource name on resources removal

Reply:
Fixed.